File: Euler-Particles/src/color/HSB/BottomColorGenerator.java
package color.HSB;

import processing.core.PApplet;

public class BottomColorGenerator extends HSBColorGenerator {

    public BottomColorGenerator(PApplet p) {
        super(p);
    }

    public int mapColor(float value, float minValue, float maxValue, int saturation, int brightness) {
        p.colorMode(PApplet.HSB, 360);
        int hue = (int) PApplet.map(value, minValue, maxValue, 0, 180);
        return p.color(hue, saturation, brightness);
    }

}

File: docs/Acceleration/color/hsb/bottom_color_generator.js
class BottomColorGenerator extends HSBColorGenerator {

    mapColor(value, minValue, maxValue, saturation, brightness) {
        colorMode(HSB, 360);
        let hue = int(map(value, minValue, maxValue, 0, 180));
        return color(hue, saturation, brightness);
    }
}
